Resources/public/js/views/services/ez-dashboardblockbasedviewservice.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockbasedviewservice', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the dashboard block based view service class
+     *
+     * @method ez-dashboardblockbasedviewservice
+     */
+    Y.namespace('eZ');
+
+    var DEFAULT_ERROR_IDENTIFIER = 'dashboardblockbasedviewservice-error',
+        DEFAULT_ERROR_MESSAGE = 'DashboardBlockBasedViewService: error';
+
+    /**
+     * The dashboard block based view service
+     *
+     * @namespace eZ
+     * @class DashboardBlockBasedViewService
+     * @constructor
+     * @extends eZ.ViewService
+     */
+    Y.eZ.DashboardBlockBasedViewService = Y.Base.create('dashboardBlockBasedViewService', Y.eZ.ViewService, [], {
+        _load: function (next) {
+            this._loadRootLocation()
```

Comment:
I don't agree. If you want to fetch all content then you'll have to write it over and over again. This is basic information and it has to be stored here. For multiple blocks you'll have to run it for each block separately. It's better to load it once and keep it here.